_posts/2017-12-27-cheap-docker-swarm-cluster-aws-terraform.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title:  "Build a Cheap Docker Swarm Cluster using Terraform and AWS"
+date:   2017-12-25 12:00:00
+author:
+  display_name: Ryan Sadler
+  login: sadler
+  email: sadler@mysticcoders.com
+  url: https://www.mysticcoders.com
+comments: true
+---
+![screenshot](/images/2017-12-27-docker-horizontal-large.png)
+
+I set out to build  a super cheap Swarm Cluster for a personal project.  With a little help from terraform you can be up and running with a multi-node swarm cluster in a matter of minutes. We aren’t going to get into what you can do with your cluster yet, that will be a future article.  We’re just going to look at prices and see how cheap we can do this.
```

Comment:
I do not like this introduction. Write a thesis statement. Write a summary. Tell the readers what they're going to get from this; this intro makes some assumptions that I find leaves the article's intent unclear.

"In this article, I'd like to show you how I built an inexpensive Swarm cluster for a personal project. I worked out a comparison between a local deployment and deployment on AWS, and found AWS to be cheaper, even though that might not be obvious on the surface; the good news is that using `terraform` meant that I could deploy my cluster in a matter of minutes."